warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__include_tts.sql

```diff
@@ -16,8 +16,8 @@ tts_issue_feeds AS (
        calitp_deleted_at
     FROM dim_stops
     -- When there is no tts_stop_name field for a given stop_name, or tts_stop_name is identical to stop_name, we proceed to run a few tests
-    WHERE (tts_stop_name IS null OR tts_stop_name = stop_name) AND
-         (
+    WHERE (tts_stop_name IS null OR tts_stop_name = stop_name)
```

Comment:
Thanks for these changes - I agree with them but just curious - are these the result of linting or a manual check?

Reply:
Just linting!

Reply:
I gotta start linting